src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,21 @@
+package seedu.addressbook.data.person;
+
+
+// Creates a Block to take in Block variable
+public class Block {
+
+ private static String block;
```

Comment:
Indentation problem.. I don't think this attribute need to be declared as `static`.